README.md

```diff
@@ -78,3 +78,41 @@ It's also possible to skip module installation altogether, giving the module dev
 ```ruby
 configure_beaker(modules: nil)
 ```
+
+# Environment variables to facts
```

Comment:
I think this heading needs to be a bit smaller:
```suggestion
## Environment variables to facts
```

Reply:
Then we need to make more changes - Modules is a feature at the same level. The second indentation level is used for the 3 options under Modules.
What you are suggesting is `# Customizing`. I'll submit that as a separate PR.